.github/workflows/tf-aws-test.yml

```diff
@@ -39,21 +42,45 @@ jobs:
     steps:
       - name: Checkout Repository
         uses: actions/checkout@v3
+
       - name: Setup Node.js v18
         uses: actions/setup-node@v3
         with:
-          node-version: 18
+          node-version: ${{ env.NODE_VERSION }}
+
+      ##################### install a local wing version #########################
+      - name: Download Dist Artifacts
+        if: ${{ env.LOCAL_BUILD ==  'true' }}
+        uses: actions/download-artifact@v3
+        with:
+          name: dist
+          path: dist
+
+      - name: Install Wing from Dist
+        if: ${{ env.LOCAL_BUILD ==  'true' }}
+        run: |
+          mkdir temp
+          cd temp
+          npm init --yes
+          npm install ../dist/*-[0-9]*.[0-9]*.[0-9]*.tgz
+          ./node_modules/.bin/wing --version
+
+      #################### install a remote wing version ########################
       - name: Install winglang globally
+        if: ${{ env.LOCAL_BUILD == 'false' }}
         uses: nick-fields/retry@v2
         with:
           max_attempts: 3
           retry_on: error
           timeout_minutes: 5
           command: npm install -g winglang
+
       - name: Installing external js modules
+        if: ${{ env.LOCAL_BUILD == 'false' }}
```

Comment:
This needs to happen regardless of LOCAL_BUILD, right? We aren't installing the required deps in any other step.